### PR TITLE
Added readme

### PR DESCRIPTION
**Pull request by johnk-Puregym**

### CODE REVIEW

File: BasketTest.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+## What is the Task?
+The coding exercise is the second stage of our technical interview process. We want it to be as real world as possible so research and use the resources necessary. 
+
+Feel free to take as long as you feel you need to on this task, we understand that not everyone has a huge amount of time while others feel they want to spend longer to demonstrate everything they can. 
+
+Note your decisions, constraints and thoughts in a readme to help us understand your process.
+
+## Why do we ask for the Task?
+
+The coding exercise allows us to view your work in the most realistic way we can giving a chance for you to show us your problem solving skills. There are many approaches to the task and being able to talk over the approach, options and considerations in the final stage of our interview.
```
Comment (robselwaypg) [inline BasketTest.md]:
```suggestion
The coding exercise allows us to view your work in the most realistic way we can, giving you a chance to show us your problem solving skills. There are many approaches to the task. Being able to talk over the approach, options and considerations in the final stage of our interview helps us understand your thought process.
```

Comment (robselwaypg) [reply BasketTest.md]: Do we suggest that the candidate will get through to the final stage here, regardless of standard of basket test?

File: BasketTest.md [hunk 2]
```diff
@@ -111,6 +122,5 @@ When submitting the task please also provide an estimate of how long you feel yo
 You can submit the test in the following ways:
 - From a shared link such as drop box, one drive, google drive etc. (preferred)
```
Comment (robselwaypg) [inline BasketTest.md:134]: Is this still our preferred? I often find `git clone` easier than a zip

File: ReadMe.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# PureGym Software Engineer Recruitment
+
+Thank you in your looking at PureGym Recruitment repo. This repo contains the exercises used in the recruitment process.
```
Comment (robselwaypg) [inline ReadMe.md]:
```suggestion
Thank you for looking at PureGym's Recruitment repo. This repo contains the exercises used in the recruitment process.
```

File: ReadMe.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+If you are looking for more information on our open vacancies, tech stack or wish to apply please see our [StackOverflow Profile](https://stackoverflow.com/jobs/companies/puregym).
+
+## Our Recruitment Process
+Our interview process consists of several stages. After the initial CV review we arrange a 30 minute call to talk about the role and for us to learn more about your skills.
+
+For most roles the next stage is a [coding exercise](BasketTest.md) with a set of scenarios and ask you to send us create your solution to the problem. This is a chance for you to demonstrate design patterns, testing code to solve a real-world problem.
```
Comment (robselwaypg) [inline ReadMe.md]:
```suggestion
For most roles the next stage is a [coding exercise](BasketTest.md) with a set of scenarios. This is a chance for you to demonstrate design patterns, testing code to solve a real-world problem.
```

File: BasketTest.md (hunk 2 above)
Comment (johnk-Puregym) [reply BasketTest.md:134]: True, no hard with people seeing other peoples versions.

**Review by darren-mcdonald — APPROVED** [review]

**Review by robselwaypg — APPROVED** [review]

**Pull request merged**